debezium-engine-app/README.md

```diff
@@ -0,0 +1,49 @@
+# Java App to Get Change Records From YugabyteDB using Debezium Engine
+
+This repository contains a Java App that uses [Debezium Engine](https://debezium.io/documentation/reference/stable/development/engine.html) and [YugabyteDB connector](https://github.com/yugabyte/debezium-connector-yugabytedb/tree/main) to stream change records from Yugabyte DB and print them.
+
+## Running the App
+
+Here are the steps to run this app
+
+1. Start YugabyteDB
+    ```
+    ./bin/yugabyted start --advertise_address <Your-IP> --ui false
+    ```
+2.  Create a table
+    ```
+    CREATE TABLE test (id int primary key, name text);
+    ```
+3. Create a stream ID using yb-admin
+    ```
+    ./yb-admin --master_addresses <Your-IP> create_change_data_stream ysql.<namespace>
+    ```
+4. Setup S3 Repository access
```

Comment:
Are we sure we need S3 credentials? Last I remember, we can compile things by downloading the jars from S3 anonymously.